examples/seismic/model.py

```diff
@@ -98,6 +150,89 @@ def demo_model(preset, **kwargs):
         return Model(vp=v, origin=origin, shape=v.shape,
                      spacing=spacing, nbpml=20)
 
+    elif preset.lower() in ['marmousitti', 'marmousi2dtti']:
```

Comment:
based on the convention you used for marmousi 3d, you should have here:

``marmousi-tti marmousi-tti2d marmousi2d-tti``